docs/pipelines/multi-account/index.md

```diff
@@ -1,11 +1,46 @@
-# Deploying Multi-Account Pipelines
+# Multiple Infrastructure-Live Repos
 
-<!-- Placeholder for future content for Pipelines v2 -->
+We recommend using a single `infrastructure-live` repository for managing your organization's infrastructure.
+Sometimes, this isn't possible due to team structure, security requirements, or other limitations.
+In order to accommodate multiple infrastructure repositories, Gruntwork Pipelines is configurable.
```

Comment:
+1 on clarifying that you should start with a single infra-live repo, but it'd be helpful to expand on when you should consider more than one infra-live repo. In particular, what are the tradeoffs? In what scenarios would multiple infra-live repos be a reasonable choice?